pymc3/distributions/arimastan.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+
+import pymc3 as pm
+
+
+def invchi(x, chisquare):
+    return (1.0 / x ** 2) * chisquare.logp(1 / x)
+
+
+class SARIMA:
+    def __init__(
+        self,
+        n,
+        n1,
+        p,
+        d,
+        q,
+        pp,
+        dd,
+        qq,
+        period,
+        d1,
+        sigma0,
+        mu0,
+    ):
+        # Model data
+        self.n = n  # number of data items
+        self.n1 = n1  # number of data items
+        self.p = p  # number of predictors  ar
+        self.d = d  # number of simple differences
+        self.q = q  # number of predictions ma
+        self.pp = pp  # number of predictors  arch
+        self.dd = dd  # number of seasonal differences
+        self.qq = qq  # number of predictions GARCH
+        self.period = period  # time series period
+        self.d1 = d1  # number of independent variables
+        self.sigma0 = sigma0  # Variance parameter
+        self.mu0 = mu0  # location parameter
+
+        # data
+
+        self.y = np.array(self.n1)  # outcome time series
```

Comment:
Is `self.n1` an integer? If so, this'll be zero-dimensional. If you want to make an array of `n1` zeros that you'll fill in later, you could do `np.zeros(self.n1)`